components/Badges.jsx

```diff
@@ -111,6 +111,13 @@ class Badges extends React.PureComponent {
       >
         <div className={this.props.classes.profileBadgeBugHunter}/>
       </Tooltip>}
+      {this.props.getSetting('displayDeveloper', true) && (this.state.flags & UserFlags.VERIFIED_DEVELOPER) !== 0 &&
```

Comment:
Setting name is too generic, maybe consider "displayBotDeveloper" or even go full verbose with "displayVerifiedBotDeveloper". It'd be pain in the ass if this causes conflicts in the future